### ISSUE TITLE
How do i use OpenEJB module with jetty 9.3.3

### ISSUE THREAD

**Issue by niteshjain132** [jetty-project/jetty-ejb#1]
Is there any documentation on how to use ?

Thanks,
Nithesh

**Comment by joakime** [comment]
Sorry, this was more of an experimental project to gauge interest in this kind of layer with Jetty.

Sadly, there's been almost no interest in it (you are officially the second person to show an interest in using openejb within jetty), and as such this layer sits in this github project more as a untested proof of concept that its possible, then as any real implementation.

That being said, this is open source, so if you want to take a stab at updating the source code to make it work with Jetty 9.3, we can guide you.

In short, this builds a jetty-distribution "module" that is installed in your `${jetty.base}` and then enabled.

Eg:

``` sh
# Build jetty-ejb
$ cd /path/to/code/jetty-ejb
$ mvn clean install -Dtest=None
# Install jetty-module-openejb
$ cd /path/to/myjetty-base
$ tar -zxvf /path/to/code/jetty-ejb/jetty-module-openejb/target/jetty-module-openejb-9.2-SNAPSHOT.tar.gz .
# Enable module openejb
$ java -jar /path/to/jetty-dist/start.jar --add-to-start=openejb
# Start your server
$ java -jar /path/to/jetty-dist/start.jar
```

**Comment by james-z-repo** [comment]
![image](https://user-images.githubusercontent.com/33625419/98216118-9cfdfd00-1f83-11eb-85d4-652d9d164ddb.png)

>> (you are officially the second person to show an interest in using openejb within jetty)

I think I'm the third and my colleague is the fourth

**Comment by joakime** [comment]
This codebase is ancient now.
It only worked with Jetty 9.2.2 against an (now EOL) version of OpenEJB.

This project is very low priority for us.
If the community is interested in this project, and provides patches, we'll review them for inclusion here.
